VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.spec.ts

```diff
@@ -0,0 +1,245 @@
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from '../logging/logger.service';
+
+import { IHearingControlsState, IHearingControlStates, VideoControlCacheService } from './video-control-cache.service';
+
+describe('VideoControlCacheService', () => {
+    let sut: VideoControlCacheService;
+    let loggerSpy: jasmine.SpyObj<LoggerService>;
+
+    beforeEach(() => {
+        loggerSpy = jasmine.createSpyObj<LoggerService>('LoggerService', ['warn', 'info']);
+        sut = new VideoControlCacheService(loggerSpy);
+
+        window.localStorage.clear();
+    });
+
+    it('should be created', () => {
+        expect(sut).toBeTruthy();
```

Comment:
is this test necessary, wouldn't this be covered by other tests using the property "sut" ?
I recommend just removing this test